webapp/tests/model/test_eligibility_data.py

```diff
@@ -544,6 +569,132 @@ def test_if_alimony_valid_and_user_a_has_elster_account_account_no_then_raise_no
             self.fail("SingleUserNoElsterAccountEligibilityData.parse_obj should not raise validation error")
 
 
+class TestSingleUserElsterAccountEligibilityData:
+
+    def test_if_alimony_valid_and_user_a_has_elster_account_no_then_raise_validation_error(self):
+        non_valid_data = {'user_a_has_elster_account_eligibility': 'no'}
+        with patch('app.model.eligibility_data.AlimonyEligibilityData.parse_obj'), \
+                pytest.raises(ValidationError):
+            SingleUserElsterAccountEligibilityData.parse_obj(non_valid_data)
+
+    def test_if_alimony_invalid_and_user_a_has_elster_account_yes_then_raise_validation_error(self):
+        non_valid_data = {'user_a_has_elster_account_eligibility': 'yes'}
+        with patch('app.model.eligibility_data.AlimonyEligibilityData.parse_obj',
+                   MagicMock(side_effect=ValidationError([], AlimonyEligibilityData))), \
+                pytest.raises(ValidationError):
+            SingleUserElsterAccountEligibilityData.parse_obj(non_valid_data)
+
+    def test_if_alimony_valid_and_user_a_has_elster_account_yes_then_raise_no_validation_error(self):
+        valid_data = {'user_a_has_elster_account_eligibility': 'yes'}
+        with patch('app.model.eligibility_data.AlimonyEligibilityData.__init__',
+                   MagicMock(return_value=None)):
+            SingleUserElsterAccountEligibilityData.parse_obj(valid_data)
+
+
+class TestElsterRegistrationMethodSoftwareEligibilityData:
+
+    def test_if_single_elster_account_valid_and_registration_method_not_software_then_raise_validation_error(self):
+        non_valid_data = {'elster_registration_method_eligibility': 'none'}
```

Comment:
This could be anything other than software, correct? I got confused at first because I thought this is looking for none. Maybe rename it 🙈?
```suggestion
        non_valid_data = {'elster_registration_method_eligibility': 'NOT_SOFTWARE'}
```

Or (I am really sorry about that recommendation) we rename the test to reflect that it is none 🙈